### ISSUE TITLE
Doesn't work on Node.js 4.3

### ISSUE THREAD

**Issue by sindresorhus** [vdemedes/gifi#2]
Works fine on latest v5.

```
❯ gifi install chalk
/usr/local/lib/node_modules/gifi/cli.js:42
    gif.kill();
       ^

TypeError: Cannot read property 'kill' of undefined
    at ChildProcess.<anonymous> (/usr/local/lib/node_modules/gifi/cli.js:42:5)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:172:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
```

**Comment by lvarayut** [comment]
I'm having the exact same error on v5.6.0.

**Comment by vadimdemedes** [comment]
Such a silly bug: https://github.com/vdemedes/gifi/commit/0c8b68adea43d5989e0303ec2f342093c72d60f9

**Comment by vadimdemedes** [comment]
Fixed and released on npm as `gifi@1.1.0`. Thanks for reporting!
